.circleci/config.yml

```diff
@@ -886,7 +886,7 @@ steps-tests: &steps-tests
         command: |
           cd src
           export ELECTRON_OUT_DIR=Default
-          (cd electron && node script/yarn test -- --ci --enable-logging)
+          (cd electron && node script/yarn test -- --ci --enable-logging --enable-features=NetworkServiceNotSupported)
```

Comment:
The feature name is weird because the default values are flipped in upstream.

Reply:
Are we planning to run the tests both with & without the network service? If we're only going to run tests with network service, let's just enable the feature.

Reply:
Yeah after working a bit on this PR, I am inclined to do that. I will adjust the PR to enable the feature by default.